tests/requirement/test_requirement_lookup.py

```
@@ -0,0 +1,278 @@
+    # minimal structural check
+    assert set(data.keys()) == set(payload.keys())
+
+
+def test_lookup_get_all_404_on_missing_token(client, monkeypatch):
+    # Arrange: make store.get_all raise ValueError (your store already does this)
+    from endoreg_db.services.lookup_store import LookupStore
+
+    orig_get_all = LookupStore.get_all
+
+    def raising_get_all(self):
+        raise ValueError(f"Lookup token {self.token} has expired or is invalid")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)